### PR TITLE
Cleanup in remote device manager and logged core

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

File: parsec/core/logged_core.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     RemoteDevicesManagerBackendOfflineError,
     RemoteDevicesManagerNotFoundError,
 )
-from parsec.core.mountpoint import mountpoint_manager_factory
+from parsec.core.mountpoint import mountpoint_manager_factory, MountpointManager
```
Comment (AntoineDupre) [inline parsec/core/logged_core.py:40]: `MountpointManager` is exposed in `parsec.core.mountpoint`

Comment (touilleMan) [reply parsec/core/logged_core.py:40]: I don't understand you point...

Comment (AntoineDupre) [reply parsec/core/logged_core.py:40]: (sorry, i made a typo) :
`MountpointManager` can not be imported directly from `parsec.core.mountpoint` but from `parsec.core.mountpoint.manager`

I fixed it:
https://github.com/Scille/parsec-cloud/pull/1415/commits/2f58d31a9a861e83b23c70472fd86c1f6c8a1906

**Review by AntoineDupre — APPROVED** [review]

**Comment by touilleMan** [comment]
Can you also merge it ?

Le lun. 31 août 2020 à 10:14, Antoine Dupré <notifications@github.com> a
écrit :

**Pull request merged**